src/services/codefixes/codeFixProvider.ts

```diff
@@ -0,0 +1,53 @@
+/* @internal */
+namespace ts {
+    export interface CodeFix {
+        name: string;
+        errorCodes: string[];
+        getCodeActions(context: CodeFixContext): CodeAction[];
+    }
+
+    export interface CodeFixContext {
+        errorCode: string;
+        sourceFile: SourceFile;
+        span: TextSpan;
+        checker: TypeChecker;
+        newLineCharacter: string;
+    }
+
+    export namespace codeFix {
+        const codeFixes: Map<CodeFix[]> = {};

```

Comment:
Forcing users to have to specify which fixes they want for each project seems counter intuitive. They can always ignore, some/all of them.... this is the behavior C#/VB has.

If we make this extensible at some point, we can revisit this.